pkg/testrunner/util.go

```diff
@@ -42,23 +42,28 @@ func GetArgoURLFromHost(host string, tr *tmv1beta1.Testrun) string {
 }
 
 // GetArgoHost returns the host of the argo ui
-func GetArgoHost(tmClient client.Client) (string, error) {
-	return GetHostURLFromIngress(tmClient, client.ObjectKey{Name: "argo-ui", Namespace: "default"})
+func GetArgoHost(ctx context.Context, tmClient client.Client) (string, error) {
+	return GetHostURLFromIngress(ctx, tmClient, client.ObjectKey{Name: "argo-ui", Namespace: "default"})
 }
 
 // GetGrafanaURLFromHostForWorkflow returns the path to the logs in grafana for a whole workflow
-func GetGrafanaURLFromHostForWorkflow(host string, tr *tmv1beta1.Testrun) string {
-	return fmt.Sprintf(`%s/explore?left=["now-7d","now","Loki",{"expr":"{container%%3D\"main\",argo_workflow%%3D\"%s\"}"},{"mode":"Logs"},{"ui":[true,true,true,"exact"]}]`, host, tr.Status.Workflow)
+func GetGrafanaURLFromHostForWorkflow(host string, workflowName string) string {
+	return fmt.Sprintf(`%s/explore?left=["now-7d","now","Loki",{"expr":"{container%%3D\"main\",argo_workflow%%3D\"%s\"}"},{"mode":"Logs"},{"ui":[true,true,true,"exact"]}]`, host, workflowName)
```

Comment:
Maybe make a constant out of `7d` and set it to `3d` as workflows older >2d shouldn't be present in the system anyway.

Reply:
thats true for most of our pods.
But I think 3d is totally ok.
For the tm dashboard we could also think if directly enter the start and end time of the needed resource. But just a future improvement